frontend/package.json

```diff
@@ -17,13 +17,19 @@
     "precommit": "lint-staged"
   },
   "dependencies": {
+    "@ledgerhq/devices": "^7.0.0",
```

Comment:
_(arbitrary comment location to start a thread)_

If I click on my address, it shows the below. We should probably show nothing / make that not clickable, and later make it clickable as part of https://github.com/ScopeLift/umbra-protocol/issues/315

<img width="319" alt="image" src="https://user-images.githubusercontent.com/17163988/193698606-508a844d-2192-470f-9112-4dd279f4564b.png">

Reply:
That makes sense will do

Reply:
Resolved

Reply:
Nit, still looks like hovering over it shows the "cursor" style pointer and not the default one. Other than that this PR LGTM!

Reply:
Good catch! I pushed up a fix